### PR TITLE
Add Python tests for ANTSRegistration

### PR DESCRIPTION
**Pull request by dzenanz**

### CODE REVIEW

File: wrapping/test/PythonANTSRegistrationTest.py [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+
+fixed_image = itk.imread(args.fixed_image)
+moving_image = itk.imread(args.moving_image)
+
+PixelType = itk.template(fixed_image)[1][0]
+Dimension = fixed_image.GetImageDimension()
+ImageType = itk.Image[PixelType, Dimension]
+assert ImageType == type(fixed_image)
+
+# registration_method = itk.ANTSRegistration[ImageType, ImageType].New(fixed_image, moving_image)
```
Comment (dzenanz) [inline wrapping/test/PythonANTSRegistrationTest.py]: Uncommenting this line causes:
```log
Loading ANTsWasm... done
Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1551, in set_inputs
    new_itk_object.SetInput(setInputNb, arg)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'itkANTSRegistrationDISS3' object has no attribute 'SetInput'. Did you mean: 'GetInputs'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1570, in set_inputs
    raise TypeError("Object accepts only 1 input.")
TypeError: Object accepts only 1 input.
python-BaseException

Process finished with exit code 1
```
and so does attempt to use it via procedural interface:
```py
forward_transform, inverse_transform = itk.ants_registration(
    fixed_image, moving_image, initial_transform=initial_transform)
```

File: wrapping/test/PythonANTSRegistrationTest.py [hunk 2]
```diff
@@ -37,8 +37,8 @@
 ImageType = itk.Image[PixelType, Dimension]
 assert ImageType == type(fixed_image)
 
-# registration_method = itk.ANTSRegistration[ImageType, ImageType].New(fixed_image, moving_image)
-registration_method = itk.ANTSRegistration[ImageType, ImageType].New()
+# registration_method = itk.ANTSRegistration[ImageType, ImageType, itk.D].New(fixed_image, moving_image)
```
Comment (dzenanz) [inline wrapping/test/PythonANTSRegistrationTest.py:40]: Uncommenting this line causes:
```log
Loading ANTsWasm... done
Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1551, in set_inputs
    new_itk_object.SetInput(setInputNb, arg)
    ^^^^^^^^^^^^^^^^^^^^^^^
AttributeError: 'itkANTSRegistrationDISS3' object has no attribute 'SetInput'. Did you mean: 'GetInputs'?

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Dev\ITK-py11\Wrapping\Generators\Python\itk\support\extras.py", line 1570, in set_inputs
    raise TypeError("Object accepts only 1 input.")
TypeError: Object accepts only 1 input.
python-BaseException

Process finished with exit code 1
```
and so does attempt to use it via procedural interface:
```py
forward_transform, inverse_transform = itk.ants_registration(
    fixed_image, moving_image, initial_transform=initial_transform)
```

Comment (dzenanz) [reply wrapping/test/PythonANTSRegistrationTest.py:40]: Here is an unsuccessful attempt to remedy this:
https://github.com/dzenanz/ANTsWasm/commit/4ec9804d918b1b119f402419063979dcadbf3580

**Review by thewtex — APPROVED** [review]
:100: excellent!

**Pull request merged**